class.cmb-meta-box.php

```diff
@@ -478,6 +478,11 @@ function save( $post_id = 0 ) {
 			return $post_id;
 		}
 
+		// Verify this meta box is for the right post type
+		if ( ! in_array( get_post_type( $post_id ), (array) $this->_meta_box['pages'] ), true ) {
```

Comment:
@swissspidy you added the strict parameter outside of the `in_array` wrap causing a 500.

Should be:
`if ( ! in_array( get_post_type( $post_id ), (array) $this->_meta_box['pages'] , true ) )`